src/terminal.rs

```diff
@@ -1,55 +1,71 @@
-#[cfg(feature="v0_48")]
 use std::path::PathBuf;
 use std::ptr;
 
 use ffi;
 use gdk;
 use glib::translate::*;
-#[cfg(feature="v0_48")]
```

Comment:
You mean these changes? Both of these imports are used in the `spawn_sync` method I added below, so they're used whether or not the feature is enabled.